app/controllers/schools/advice/electricity_intraday_controller.rb

```diff
@@ -2,6 +2,10 @@ module Schools
   module Advice
     class ElectricityIntradayController < AdviceBaseController
       def insights
+        @peak_usage_benchmarking = build_peak_usage_benchmarking
+        @peak_usage_calculation = build_peak_usage_calculation(asof_date)
+        @peak_usage_calculation_1_year_ago = build_peak_usage_calculation(previous_years_asof_date)
```

Comment:
this looks like it could fail if the school doesn't have >1 years of data. So I'd add a check for this in the code and return nil from the calculation. The template can then conditionally include the percentage.